src/main/java/design/MazeDesignerInputBoundary.java

```diff
@@ -0,0 +1,18 @@
+package design;
+
+public interface MazeDesignerInputBoundary {
+    void newMaze();
+
+    void resetMaze();
+
+    void buildWall(int row, int col);
+
+    void removeWall(int row, int col);
+
+    void startPoint(int row, int col);
+
+    void endPoint(int row, int col);
+
+    void randomMaze();
```

Comment:
please add the appropriate access modifiers for these functions (public/private)